plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -122,8 +129,8 @@ const fetchPluginData = async ({createNode, reporter, firstReleases}) => {
                     wiki: pluginData.wiki,
                     securityWarnings: updateData.warnings.filter(p => p.name == pluginName)
                         .map(w => checkActive(w, pluginUC)),
-                    dependencies: pluginData.dependencies,
-                    categories: pluginData.categories,
+                    dependencies: pluginUC.dependencies.concat(getImpliedDependencies(plugin, detachedPlugins, updateData)),
+                    categories: pluginUC.labels.map(label => labelToCategory[label]).filter(unique),
```

Comment:
Alternatively:
Array.from(new Set(pluginUC.labels.map(label => labelToCategory[label])))

Reply:
if we are just mapping labels to categories, we don't really need categories in plugins at all do we? Couldn't it just be a graphql query on the displaying plugin page?

Reply:
kindof, it would also need some refactoring for the Algolia index I suppose

Reply:
actually the search index is the only place where we need a connection between plugins and categories.

Reply:
oh interesting, so we could in theory just say hey algoila, we want anything with labels x y and z and not worry about linking it to a specific plugin? thats cool

Reply:
done in the last commit. So, if the whole category thing is just to define grouping of labels in the search form, maybe it's OK to keep the category definition in this repo?